cmd/minishift/cmd/addon/addon_enable.go

```diff
@@ -38,7 +38,7 @@ var addonsEnableCmd = &cobra.Command{
 }
 
 func init() {
-	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on when it is applied.")
+	addonsEnableCmd.Flags().IntVar(&priority, priorityFlag, 0, "The priority of the add-on. Determines the order in which to apply the add-on.")
```

Comment:
Passive voice usage isn't recommended (tech writing 101, let's talk about it someday), and the meaning is basically the same so I don't see why we need to rephrase it.